common/db_test.go

```diff
@@ -656,6 +659,83 @@ func TestLoadWinningTickets_GivenTwoSessionsWithTickets_ReturnsAllTickets(t *tes
 	assert.Equal(recipientRand1, recipientRands[1])
 }
 
+func TestInsertMiniHeader_ReturnsFindLatestMiniHeader(t *testing.T) {
+	dbh, dbraw, err := TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	assert := assert.New(t)
+	require := require.New(t)
+	require.Nil(err)
+
+	h0 := defaultMiniHeader(t)
+	err = dbh.InsertMiniHeader(h0)
```

Comment:
Test the error paths for `InsertMiniHeader()`?

Reply:
Fixed in ced063e